packages/bitcore-wallet-client/test/api.test.js

```diff
@@ -7042,12 +7042,75 @@ describe('client API', function() {
                               recoveryClient.getBalance({}, (err, b) => {
                                 balance = b.totalAmount;
                                 next(err);
-                              });
+                              }, '/v1/balance/');
                             }, 200);
                           },
                           err => {
                             should.not.exist(err,err);
-                            balance.should.equal(1e8);
+                            balance.toString().should.equal("100000000");
```

Comment:
the "toString()" here kind of defeat the purpose of the test. Maybe you can check that typeof balance is number?